easybuild/easyconfigs/c/Commet/Commet-20160422-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'MakeCp' 
+
+name = 'Commet' 
+version = '20160422' 
+
+homepage = 'https://colibread.inria.fr/software/commet/' 
+description = """ COMMET (“COmpare Multiple METagenomes”) provides a global similarity overview between all datasets of a large metagenomic project. 
```

Comment:
also, please replace the unicode `"` characters with regular ones, while you're at it